src/games/blackjack/index.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  Transaction as TransactionType,
+  TransactionObjectArgument,
+  TransactionArgument,
```

Comment:
The blackjack file should include the different command types like hit, split, doubel down etc.